docs/user/environment-variables.md

```diff
@@ -38,6 +38,8 @@ The installer accepts a number of environment variable that allow the interactiv
 
 ## Platform-Specific
 
+* `AWS_PROFILE`:
```

Comment:
If the `OPENSHIFT_INSTALL_` prefix was used here, as with the other variables, it'd be consistent and also allow for a different profile used with the openshift-installer than what is generally set for the environment. The paid for that is to override `AWS_PROFILE` accordingly before calling out to terraform in the installer and also pass it to the aws-sdk-go in the code somehow.

Reply:
> If the `OPENSHIFT_INSTALL_` prefix was used here...

I prefer `AWS_PROFILE` as it stands because it's a widely-used variable.  One less thing for users to learn about before calling the installer.

> ... also allow for a different profile used with the openshift-installer than what is generally set for the environment.

Do we expect this to be frequent?  As it stands, folks can override for just the installer:

```console
$ AWS_PROFILE=whatever openshift-installer
```

Although #336 makes all of this moot at the moment ;).

Reply:
:) looking!

Reply:
from aws-sdk-go docs: 
`By default, the SDK checks the AWS_PROFILE environment variable to determine which profile to use. If no AWS_PROFILE variable is set, the SDK uses the default profile.`

Reply:
more info, I unset AWS_PROFILE to get the exact error: when i run with this PR:
```DEBUG       Generating Tectonic Manifests...       
FATAL Failed to generate Terraform Variables: failed to fetch dependency for Terraform Variables: failed to fetch dependency for Bootstrap Ignition Config: failed to fetch dependency for Common Manifests: failed to generate asset Tectonic Manifests: SharedCredsLoad: failed to get profile
caused by: section 'default' does not exist 
```

Reply:
> Although #336 makes all of this moot at the moment ;).

Ah that's unfortunate, I did expect this to be passable as an argument to the aws-go-sdk functions. After some digging there's [a way to provide credentials](https://docs.aws.amazon.com/sdk-for-go/api/aws/#Config.WithCredentials) to the SDK.
Although after thinking about this:

> Do we expect this to be frequent? As it stands, folks can override for just the installer: ...

I think the effort is probably not worth it and we can document the usage of `AWS_PROFILE` somewhere.